plugins/net_plugin/net_plugin.cpp

```diff
@@ -632,7 +632,6 @@ namespace eosio {
       out_queue.clear();
       if(response_expected) {
         response_expected->cancel();
-        response_expected.reset();
```

Comment:
This was added by: PR #831 .

Should it instead be:
`response_expected.reset(new boost::asio::steady_timer(app().get_io_service()));`

Reply:
no, I am simply not getting rid of the current timer instance.